skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.kt

```diff
@@ -136,6 +136,10 @@ internal class MetalRedrawer(
         )
     }
 
+    override fun setVisible(isVisible: Boolean) {
+        setLayerVisible(device, isVisible)
```

Comment:
Could you add a screenshot test to SkiaLayerTest for this?

Reply:
Yes